taichi/ir/statements.h

```diff
@@ -382,6 +444,9 @@ class GlobalLoadStmt : public Stmt {
   TI_DEFINE_ACCEPT_AND_CLONE;
 };
 
+/**
+ * A global store.
```

Comment:
Could you add a few more details, explaining that we are using this to access not only the SNodes, but also external array, TLS, BLS?

Reply:
Thanks for reminding this! I forgot it when writing this part.